node/client/Cargo.toml

```diff
@@ -29,7 +29,7 @@ sc-service = { git = "https://github.com/paritytech/substrate", branch = "master
 
 pallet-mmr-primitives = { git = "https://github.com/paritytech/substrate", branch = "master" }
 
-beefy-primitives = { git = "https://github.com/paritytech/grandpa-bridge-gadget", branch = "master" }
+beefy-primitives = { git = "https://github.com/paritytech/grandpa-bridge-gadget", branch = "bkchr-upgrade-jsonrpc" }
```

Comment:
shouldn't these be `master`?

Reply:
This was to make the companion happy. It is now master